src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+/**
+ * An example of a custom control using FXML.
+ * This control represents a dialog box consisting of an ImageView to represent the speaker's face and a label
+ * containing text from the speaker.
+ *
+ * @author Samsation
+ * @version CS2103T AY 22/23 Sem 1
+ */
+public class DialogBox extends HBox {
+    @FXML
+    private Label dialog;
+    @FXML
+    private ImageView displayPicture;
+
+    /**
+     * A constructor for DialogBox
+     *
+     * @param text The text to be displayed.
+     * @param img The display image.
+     */
+    private DialogBox(String text, Image img) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource("/view/DialogBox.fxml"));
+            fxmlLoader.setController(this);
+            fxmlLoader.setRoot(this);
+            fxmlLoader.load();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        dialog.setText(text);
+        displayPicture.setImage(img);
+    }
+
+    /**
+     * A method that flips the dialog box such that the ImageView is on the left and text on the right.
```

Comment:
Perhaps you could change your method header to start with 'Flips'? According to the provided [coding standards](https://se-education.org/guides/conventions/java/intermediate.html), method headers should start with words like 'Returns', 'Sends', 'Adds' etc

Reply:
Fixed under A-CodeQuality. Thank you!